packages/block-serialization-default-parser/parser.php

```diff
@@ -391,8 +391,8 @@ function next_token() {
 		 * are associative arrays. If we use `array()` we get a JSON `[]`
 		 */
 		$attrs = $has_attrs
-			? json_decode( $matches[ 'attrs' ][ 0 ], /* as-associative */ true )
-			: json_decode( '{}', /* don't ask why, just verify in PHP */ false );
+			? wp_json_decode( $matches[ 'attrs' ][ 0 ], /* as-associative */ true )
+			: wp_json_decode( '{}', /* don't ask why, just verify in PHP */ false );
```

Comment:
This allows to use this parser only in a WordPress context. I don't think that's what we want.